Sources/Plasma/NucleusLib/pnNetBase/pnNbConst.h

```diff
@@ -81,6 +81,7 @@ const unsigned kMaxPublisherAuthKeyLength       = 64;
 const unsigned kMaxGTOSIdLength                 = 8;
 const unsigned kMaxGameScoreNameLength          = 64;
 const unsigned kMaxEmailAddressLength           = 64;
+const unsigned kMaxTracebackLength              = 1024;
```

Comment:
It would be nice if we could just increase this value. It's an artificial limit - at the protocol level, the length is sent as a `uint16_t`. DIRTSAND and MOSS already accept longer lengths, but Cyan's server probably enforces the 1024-char limit like the client does :(